keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -858,6 +849,37 @@ in back-to-back releases.
 - Deprecate the flag
 -->
 
+#### Alpha
+
+- Initial implementation added
+- CRI implementations starting to add support
+
+#### Beta
```

Comment:
I really want this KEP to progress faster as it will help us have a good point of discussions with the model producers. However I have trouble rationalizing how we can change APIs and have Containerd support not merged and declare the feature as beta. Even if it is OFF by default.

I think we have to go to beta in 1.33 with way more confidence and keep it as a alpha in 1.32.

/hold

Reply:
+1

cc/ @samuelkarp @mikebrow

Reply:
I hear you about the API change, I still disagree on CRI support. I don't think we need to hash this out now, and we don't need to push to beta here, but I would like to come to consensus as a SIG on what the policy is going forward. It was my understanding our consensus said we'd move forward here and I'd like to avoid this confusion in the future

Reply:
We held a containerd community meeting today and 1) the general consensus/agreement was that we will definitely have our 2.0 GA release before kubecon; 2) we have consensus for heading back to roughly twice a year minor release cycles after 2.0; 3) the prior version of this feature is fairly close to merge in our main branch feeding our 2.0 release candidates .. we are still considering the best logic for the gc life cycle of the pod scoped host image volume created by the runtime from the image. 

A) I think the proposal here for beta criteria was that it would be merged into containerd main and made available as a release candidate before the beta designation would be merged into kubelet, and the feature gate would remain default off if we don't GA it in time, but could be promoted to default on if GA'd in 2.0 and there is consensus by sig-node owners. 
B) We still need the other changes in this KEP update.

Reply:
Those are great news!

We still need to iterate on API and make sure containerd has the implementation merged. Not promoting to beta is a statement of confidence from k8s standpoint.

Reply:
I commented on the thread in the SIG architecture mailing list [https://groups.google.com/d/msgid/kubernetes-sig-architecture/99da48f1-b9f5-47d8-8730-a17e87acd11bn%40googlegroups.com](https://groups.google.com/d/msgid/kubernetes-sig-architecture/99da48f1-b9f5-47d8-8730-a17e87acd11bn%40googlegroups.com?utm_medium=email&utm_source=footer)  and raising my concern here for visibility too

This feature adds a new field to an stable object, like the Pod, and that has a lot of implications and consequences,
The user facing problem is also , anecdotally, evident as you can see here https://github.com/kubernetes-sigs/kind/issues/3745